source/ns_cmdline.c

```diff
@@ -1110,6 +1110,12 @@ static int cmd_run(char *string_ptr)
     int argc, ret;
 
     tr_info("Executing cmd: '%s'", string_ptr);
+
+    // Initialize enough of variables to utilize the existing command-not-found -path in case of
+    // getting only whitespace(s) as command string.
+    cmd.cmd_ptr = NULL;
```

Comment:
I don't think this is absolutely necessary, but doesn't hurt anything either.

Reply:
Right you are, removed.